prov/mrail/src/mrail_av.c

```diff
@@ -92,26 +92,31 @@ static int mrail_av_insert(struct fid_av *av_fid, const void *addr, size_t count
 					"addr", addr);
 			ret = fi_av_insert(mrail_av->avs[j],
 					   (char *)addr + offset, 1,
-					   NULL, flags, NULL);
+					   &index, flags, NULL);
 			if (ret != 1) {
 				free(peer_info);
 				return ret;
 			}
 			offset += mrail_av->rail_addrlen[j];
+			if (j == 0)
+				index0 = index;
```

Comment:
nit: can index0 be renamed to index_rail0?

Reply:
sure.